doppler/resource_secret.go

```diff
@@ -92,6 +113,21 @@ func resourceSecretRead(ctx context.Context, d *schema.ResourceData, m interface
 		return handleNotFoundError(err, d)
 	}
 
+	nilFields := []string{}
+	if secret.Value.Raw == nil {
+		nilFields = append(nilFields, "raw")
+	}
+	if secret.Value.Computed == nil {
+		nilFields = append(nilFields, "computed")
+	}
+
+	if len(nilFields) > 0 {
+		return diag.FromErr(fmt.Errorf(
+			"One or more secret fields are restricted: %v. "+
+				"Make sure that you are using a service account to perform Terraform actions. "+
```

Comment:
Should this be "service account or service token"?

Reply:
Good point, I definitely think service accounts are going to be the more common use case but service tokens are valid